doc/user/content/get-started/install-materialize.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Install Materialize locally"
+description: "Download the Docker image and run Materialize locally for evaluation"
+menu:
+  main:
+    parent: "get-started"
+    weight: 16
+    name: "Install Materialize locally"
+
+---
+
+For testing/evaluation purposes, you can download the Materialize Docker image
+and run Materilize locally in a Docker container.  The Docker image:
+
+- Includes the [Quickstart](/get-started/quickstart) data source
+
+- Supports connecting your data sources
+
+- Provides [community support via
+  Slack](https://materializecommunity.slack.com/join/shared_invite/zt-2bad5ce4i-ZsiPWI5jd7Q9pRDGYj3dkw#/shared-invite/email)
```

Comment:
Should we include the comparison table @ggnall created somewhere as an overview here, just to make it clearer? Then we can elaborate more each of the characteristics under `Limitations`.

Reply:
Yup ... but, until the text in the table is final, figured I won't include it until then.